### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/web/web-in-60`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4577]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4577|
|Type|enhancement|
|Created|2010-07-11 07:08:46Z|
|Branch|https://github.com/twisted/twisted/tree/sphinx-web60-4577|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for `projects/web/web-in-60`

Attachments:

* [web-in-60.xhtml.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/40f/40f45910f175ee8531124b7e5889358e5bcaa384/edb5bd2d630ccd613bdb12480add9b4bf35f352b.patch) (48652 bytes) - added by jdb on 2010-09-10 17:35:22Z - Adapted the lore input to fix typos in the html. Suppressed in the nested formatting.

<details><summary>Searchable metadata</summary>

```
trac-id__4577 4577
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_sphinx_web60_4577 branches/sphinx-web60-4577
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1278832126000000 1278832126000000
changetime__1284827815000000 1284827815000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="jdb's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|jdb removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [30036]) Branching to 'sphinx-web60-4577'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

(In [30037]) Apply web-in-60.xhtml.patch

refs [#4577](https://github.com/twisted/twisted/issues/4577)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

(In [30038]) Fix the spacing in the "WSGI applications" link

refs [#4577](https://github.com/twisted/twisted/issues/4577)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks.  Aside from the one extra fix I made, looks good.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [30040]) Merge sphinx-web60-4577

Author: jdb
Reviewer: exarkun
Fixes: [#4577](https://github.com/twisted/twisted/issues/4577)

Tweaks to the Lore markup in the Web documentation to make it
more amenable to conversion to Sphinx.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
